docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+ 2.  Leave an organization that he's tied to by direct Subject reference, but not one that he is tied to by a Group suibject.  That's an odd wrinkle.
+
+
+## OrgGroups Overview
+Organization owner need to be able to define groups so they can efficiently manage policy on their projects, however a organization owner is unlikely to be allowed to manage cluster-scoped group membership across all projects.  A single organization owner (or set of organization owners) is likely to own multiple related projects (think single department running multiple projects).  To allow cluster-scoped group definitions to be shared across multiple related projects, we'll introduce the concept of OrgGroups (scoped sets of users) that are logicaly owned by Organizations.
+
+OrgGroups look a lot like Groups, but they are always logically scoped to a particular Organization and they behave differently.  They will be cluster scoped resources, but we'll indicate the Organization scope of an OrgGroup by prefixing every OrgGroup name with Organization name.  This prefixing allows two organizations to have groups with the same name.  OrgGroups are managed by Organization.Owners (Groups are managed by cluster admins), OrgGroups can only contain Users that are Organization.Members (Groups can contain any user), OrgGroups can only be bound to roles in projects owned by the Organization (Groups can be bound to roles in any project).
+
+
+### Use Cases
+ 1.  I want to enable a subset of users to be able to organize sets of Users into OrgGroups, but I only want those changes to affect policy evaluation across some projects.
+
+### OrgGroups
+```go
+// OrgGroup contains the set of Users that belong to a given OrgGroup.  The name of a OrgGroup is required to 
+// to be in the form of "OrganizationName:OrgGroupName".
+type OrgGroup struct {
+	kapi.TypeMeta
+	kapi.ObjectMeta
+
+	// Users is a list of Users that are members of the OrgGroup
+	Users []string
+}
+```
+
+A reverse lookup index will be built in memory (similar to the project cache) to be allow the authenticator to make fast lookups for OrgGroup membership determination.  A valid OrgGroup.Name can never be used as a Group.Name and vice-versa.  This makes it impossible for anyone (including cluster-admins) to create a Group that will collide with an OrgGroup.
+
+
+### user.Info and OrgGroups
+user.Info is an interface shared with kubernetes.  If they won't change their interface, we'll be able to re-use it in a backwards compatible way by scope qualifying our OrgGroup names as Groups.  This means that being a member of OrgGroup "MyOrg:AOrgGroup" would result in a Group membership that looks like "org:MyOrg:AOrgGroup".  The only signficant side effect is that RoleBinding RESTStorage must reject Groups that start with "org:", but those Groups are invalid Group names anyway.
+
+
+## Tying Projects to Organizations
```

Comment:
Shouldn't you mention Users here as well, since it was stated in the first paragraph Organization xor User?